common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1238,7 +1238,10 @@ def submit_problem(self, data, override_time=False):
             # without a stack trace
             else:
                 # Translators: {msg} will be replaced with a problem's error message.
-                msg = _(u"Error: {msg}").format(msg=inst.message)
+                if type(inst) == StudentInputError:
```

Comment:
@staubina I'm OK with removing "Error" completely. The text of the message should clearly indicate its purpose.